etc/install_deps.sh

```diff
@@ -1,20 +1,14 @@
 #!/bin/bash
```

Comment:
given that this file is being tailored for travis deployment, it is better to name this as `travis_installs.sh` and keep the original `install_deps.sh` for local installations.